blog/2015/10/call-for-instructor-training.html

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog
+root: ../../..
+author: Raniere Silva
+title: "Call for Applications to December Instructor Training"
+date: 2015-10-20
+time: "22:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+As we <a href="{{page.root}}/blog/2015/09/rebooting-instructor-training.html">announced previous</a>,
+we will run another round of the free/open Instructor Training in December.
+Is time for people interested in participate at this round apply for the
+available seats.
+</p>
+<!-- end excerpt -->
+<h2>Rules</h2>
+<p>Students <strong>must</strong> apply in groups that are able to co-locate in one room for the duration of the Instructor Training. And each group should commit to jointly run one workshop no later than April 1st, 2016. <strong>Please note: how to run a workshop is covered in the curriculum.</strong></p>
+<p>Each student is expected to:</p>
+<ul>
+  <li><p>Be on the current waiting list or demonstrate sufficient competence with core tools used in the lessons they will teach.</p></li>
+<li><p>Watch one of the two videos below and submit a paragraph of feedback on the video they chose to watch as part of their application.</p>
+<ul>
+<li>https://vimeo.com/139316669</li>
+<li>https://vimeo.com/139181120</li>
+</ul></li>
+<li><p><strong>Only</strong> students interested in teaching for Software Carpentry need to submit a pull request to the specified GitHub repository.</p>
```

Comment:
I would change this to, "People who wish to teach Software Carpentry must submit a small pull request to a specified GitHub repository.  People who wish to teach Data Carpentry do not need to do so."  (And have you spoken to DC to make sure they've reviewed this doc and these rules?)